addons/nachschub/configs/CfgVehicles_USA.hpp

```diff
@@ -256,6 +256,98 @@ class TB_supply_usa_night : WRAPPER_NAME(Box_East_Support_F)
+        MACRO_ADDMAGAZINE(rhs_mag_m714_White_precise,2);
+        MACRO_ADDMAGAZINE(rhs_mag_m713_Red_precise,2);
+        MACRO_ADDMAGAZINE(rhs_mag_m715_Green_precise,2);
+
+        // HuntIR
+        MACRO_ADDMAGAZINE(ACE_HuntIR_M203,4);
+
+        // Grenades
+        MACRO_ADDMAGAZINE(SmokeShell,20);
+        MACRO_ADDMAGAZINE(SmokeShellRed,4);
+        MACRO_ADDMAGAZINE(SmokeShellBlue,4);
+
+        MACRO_ADDMAGAZINE(rhs_mag_mk3a2,20);    // Offensivenade
+        MACRO_ADDMAGAZINE(B_IR_Grenade,4);      // IR-Granate
+
+        // 5.56
+        MACRO_ADDMAGAZINE(rhs_mag_30Rnd_556x45_M855A1_EPM_Pull_Tracer_Red,40);  // Standard GewehrMag
+
+        // MGs
+        MACRO_ADDMAGAZINE(rhsusf_200Rnd_556x45_M855_mixed_soft_pouch_coyote,3); // MGMag5.56 Tracer
+        MACRO_ADDMAGAZINE(rhsusf_200Rnd_556x45_M855_soft_pouch_coyote,3);       // MGMag5.56
+        MACRO_ADDMAGAZINE(TB_mag_100Rnd_556x45_Mk318_tracer,6);                 // MGMag5.56 C-Mag Tracer
+        MACRO_ADDMAGAZINE(TB_mag_100Rnd_338_LS_Tracer,6);                       // MGMag8.6 Tracer
+        MACRO_ADDMAGAZINE(TB_mag_100Rnd_338_LS_DIM,6);                          // MGMag8.6 DIM
+        MACRO_ADDMAGAZINE(rhsusf_100Rnd_762x51_m62_tracer,6);                  // MGMag7.62 AP Tracer
+        MACRO_ADDMAGAZINE(rhsusf_100Rnd_762x51_m61_ap,6);                      // MGMag7.62 AP
+
+        // Secondary
+        MACRO_ADDMAGAZINE(rhsusf_mag_17Rnd_9x19_JHP,15);
+        MACRO_ADDMAGAZINE(rhsusf_mag_7x45acp_MHP,15);
+        MACRO_ADDMAGAZINE(rhsusf_mag_15Rnd_9x19_JHP,15);
+        MACRO_ADDMAGAZINE(TB_mag_45_FMJ,15);
+
+        // 12.7
+        MACRO_ADDMAGAZINE(rhsusf_mag_10Rnd_STD_50BMG_M33,6);
+        MACRO_ADDMAGAZINE(rhsusf_mag_10Rnd_STD_50BMG_mk211,6);
+
+        // 8.6
+        MACRO_ADDMAGAZINE(TB_mag_10Rnd_338_LS_Tracer,6);
+        MACRO_ADDMAGAZINE(TB_mag_10Rnd_338_LS_DIM,6);
+
+        // DMR7.62
+        MACRO_ADDMAGAZINE(rhs_mag_20Rnd_SCAR_762x51_m80a1_epr,8);               // SCAR TPR
+        MACRO_ADDMAGAZINE(rhsusf_5Rnd_762x51_m118_special_Mag,32);              // M24
+        MACRO_ADDMAGAZINE(rhsusf_20Rnd_762x51_SR25_m62_Mag,8);                  // DMRMag7.62 SR25
+        MACRO_ADDMAGAZINE(rhsusf_20Rnd_762x51_m993_Mag,8);                      // DMRMag7.62 M14
+
+        // Shotgun
+        MACRO_ADDMAGAZINE(rhsusf_8Rnd_00Buck,7);                                // Schrotflinte
+        MACRO_ADDMAGAZINE(rhsusf_8Rnd_Slug,7);                                  // Schrotflinte
+        MACRO_ADDMAGAZINE(rhsusf_5Rnd_00Buck,7);                                // Schrotflinte
+        MACRO_ADDMAGAZINE(rhsusf_5Rnd_Slug,7);                                  // Schrotflinte
+    };
+
+    class TransportItems
+    {
+        MACRO_ADDITEM(ACE_quikclot,50);
+        MACRO_ADDITEM(ACE_elasticBandage,50);
+
+        MACRO_ADDITEM(ACE_salineIV,4);
+
+        MACRO_ADDITEM(ACE_morphine,20);
+        MACRO_ADDITEM(ACE_epinephrine,10);
+    };
+
+    class TransportBackpack
```

Comment:
Es wäre `TransportBackpacks` statt `TransportBackpack` dass S muss mit.
Deswegen immer mit dem WRAPPER arbeiten, alleine schon damit wichtige Variablen wie isTB gesetzt sind.
Habe alles in einem Commit gefixt bitte nochmal prüfen.